C/CompilerSupportLibraries/CompilerSupportLibraries@v1.0/build_tarballs.jl

```diff
@@ -1,5 +1,5 @@
 include("../common.jl")
 
-build_csl(ARGS, v"1.0.3"; preferred_gcc_version=v"12", windows_staticlibs=true, julia_compat="1.9")
+build_csl(ARGS, v"1.0.2"; preferred_gcc_version=v"12", windows_staticlibs=true, julia_compat="1.9")
```

Comment:
Only thing, maybe we should bump the version number to 1.0.4?

Reply:
Probably...